lib/ansible/modules/cloud/misc/helm_cli.py

```diff
@@ -0,0 +1,595 @@
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli
+short_description: Manages Kubernetes packages with the Helm package manager
+description:
+  - Install, upgrade, delete packages with the Helm package manager.
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+  - Matthieu Diehr (@d-matt)
+requirements:
+  - "helm (https://github.com/helm/helm/releases)"
+  - "yaml (https://pypi.org/project/PyYAML/)"
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use, relative to the 'service_path'
+        unless you supply an absolute path.
+    required: false
+    type: path
+  chart_ref:
+    description:
+      - chart_reference on chart repository
+      - path to a packaged chart
+      - path to an unpacked chart directory
+      - absolute URL
+      - Required when release_state is set to present
+    required: false
+    type: str
+  chart_repo_url:
+    description:
+      - Chart repository url where to locate the requested chart
+    required: false
+    type: str
+  chart_repo_username:
+    description:
+      - Chart repository username where to locate the requested chart
+      - Required if chart_repo_password is specified
+    required: false
+    type: str
+  chart_repo_password:
+    description:
+      - Chart repository password where to locate the requested chart
+      - Required if chart_repo_username is specified
+    required: false
+    type: str
+  chart_version:
+    description:
+      - Chart version to install. If this is not specified, the latest version is installed
+    required: false
+    type: str
+  release_name:
+    description:
+      - Release name to manage
+    required: true
+    type: str
+    aliases: [ name ]
+  release_namespace:
+    description:
+      - Kubernetes namespace where the chart should be installed
+      - Can't be changed with helm 2
+    default: "default"
+    required: false
+    type: str
+    aliases: [ namespace ]
+  release_state:
+    choices: ['present', 'absent']
+    description:
+      - Desirated state of release
+    required: false
+    default: present
+    aliases: [ state ]
+    type: str
+  release_values:
+    description:
+        - Value to pass to chart
+    required: false
+    default: {}
+    aliases: [ values ]
+    type: dict
+  tiller_host:
+    description:
+      - Address of Tiller
+      - Ignored when is helm 3
+    type: str
+  tiller_namespace:
+    description:
+      - Namespace of Tiller
+      - Ignored when is helm 3
+    default: "kube-system"
+    type: str
+  update_repo_cache:
+    description:
+      - Run `helm repo update` before the operation. Can be run as part of the package installation or as a separate step.
+    default: "false"
```

Comment:
```suggestion
    default: false
```
`"false"` is a string, not a boolean